s3/s3_transport/unit_tests/src/test_s3_transport.cpp

```diff
@@ -1041,6 +1089,30 @@ TEST_CASE("s3_transport_readwrite_thread", "[rw][thread]")
     remove_bucket(bucket_name);
 }
 
+TEST_CASE("test_seek_end_existing_file", "[seek_end]")
+{
+    rodsLogLevel(log_level);
+
+    std::string bucket_name = create_bucket();
+
+    std::string filename = "medium_file";
+    std::string object_prefix = "dir1/dir2/";
+
+    SECTION("seek end small file")
+    {
+        filename = "small_file";
+        test_seek_end(bucket_name, filename, object_prefix, keyfile);
+
+    }
+
+    SECTION("seek end medium file")
```

Comment:
are these small and medium files already created/managed by the test framework?

Reply:
There is a script that can be run to create them.

Reply:
My mistake.  There is no README for the unit tests but the script is there.

Reply:
got it - just checking as it seemed to reference something I couldn't see.